src/System.Management.Automation/help/HelpCommentsParser.cs

```diff
@@ -493,49 +493,27 @@ private void BuildSyntaxForParameterSet(XmlElement command, XmlElement syntax, M
             command.AppendChild(syntax).AppendChild(syntaxItem);
         }
 
-        private static void GetExampleSections(string content, out string prompt_str, out string code_str, out string remarks_str)
+        internal static void GetExampleSections(string content, out string prompt_str, out string code_str, out string remarks_str)
```

Comment:
Can we use a public API for a test? Or `internal XmlDocument BuildXmlFromComments()`?

Reply:
Yes, I thought about it too.

Then I looked around and decided that the setup and assertions would be much more complicated, so overall it would not be worth it really IMO.

Reply:
Perhaps we could find a script repo and make Pester test.

Reply:
I looked around again and found https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Language/Scripting/ScriptHelp.Tests.ps1

It's weird that I didn't spot it earlier...

Extending these PS tests should work fine. I can give it a try, but probably not before next weekend.